common/core/src/main/resources/schemata/herocard-connector-response-schema.json

```diff
@@ -76,6 +76,7 @@
         "user_input": {"type": "array", "items": {"$ref": "#/definitions/CARD_ACTION_USER_INPUT"}},
         "request":    {"type": "object"}
       },
+      "required": ["id", "completed_label"],
```

Comment:
Actually, could you make this `["id", "action_key", "label", "completed_label", "url", "type"]`

https://github.com/vmwaresamples/card-connectors-guide/pull/7/files

Reply:
Added ```"id", "action_key", "label", "completed_label", "url", "type"]``` fields to required.

Reply:
I don't see them.  Did you lose the changes when pulling/rebasing or something?

Reply:
sorry.. i forgot to push the changes...  I have pushed the changes now..